GVFS/GVFS.Platform.Mac/MacFileSystemVirtualizer.cs

```diff
@@ -349,28 +349,20 @@ private bool TryGetSymLinkTarget(string sha, out string symLinkTarget)
                 string sha = GetShaFromContentId(contentId);
                 byte placeholderVersion = GetPlaceholderVersionFromProviderId(providerId);
 
-                EventMetadata metadata = this.CreateEventMetadata(relativePath);

```

Comment:
We need to undo these changes to the Mac platform, as we will want to trace this data carefully while we are still learning about the platform. @jeschu1 is on it. (I'll strip out the change in this PR, though.)